### ISSUE TITLE
Post deploy configuration of moc-infra openshift cluster

### ISSUE THREAD

**Issue by naved001** [CCI-MOC/ops-issues#1222]
This may includes:
- [x] configuration authentication - SSO
- [ ] ingress
- [x] external secrets operator
- [ ] probably metallb?  

this issue may be split up into more issues

**Comment by naved001** [comment]
I have done some configuration.

the moc-infra cluster now uses github idp, and members of ops-core team now has access to the cluster, though only certain users have been added to the sudoers rule so they can impersonate `system:admin` for privileged operations. 

I also installed the external secrets operator, I created an appropriate IAM role in AWS to use with it. 

The configuration for this cluster lives here: https://github.com/naved001/moc-infra-config/
